src/core/sys/windows/winsock2.d

```diff
@@ -740,7 +740,21 @@ struct hostent
 }
 
 // Note: These are Winsock2!!
-struct WSAOVERLAPPED;
+struct WSAOVERLAPPED
+{
+    ulong Internal;
```

Comment:
In the Windows API these are defined as `ULONG_PTR`, and so D analog is either `size_t`, or `core.stdc.stdint.uintptr_t` if you want to be pedantic about the use of ISO C terminology.

Reply:
Or perhaps better, use the Windows typedef types directly - that way there should be no room for error, or at least only one place to fix.